plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/layout/SubsystemLayout.java

```diff
@@ -51,21 +54,50 @@
   private Subscription retained; //NOPMD field due to GC
   private DataSource<?> source;
 
+  private List<Component> hidden = new ArrayList<>();
```

Comment:
Why does only `SubsystemLayout` get this behavior? I'd expect to be able to hide widgets in _any_ type of layout